### PR TITLE
Feature/general improvements 1

### LINKED ISSUE #83: Change "delete after (days)" text in meeting creation form to "delete after (days) from creation" (or something similar)
No functions will be changed. Only text

### LINKED ISSUE #86: Sort the dates by time in the meeting page
Function to be Edited:
- `show($id)` in MeetingController class. This function will include the function which would sort the received dates in ascending order by their `date_and_time` value.

### LINKED ISSUE #87: Lower the maximum amount of days to preserve a meeting from 32000 to 180
Function to be edited:
- `rules()` of StoreMeeting class. This function will have the maximum value of the `delete_after` altered from 32000 to 180

### LINKED ISSUE #88: Update the vote display on the meeting menu to also show the meeting's end time
No functions will be changed. Only text on the front end

### LINKED ISSUE #90: Date update is not working.
**Describe the bug**
Date update is not working. This is due to merging of date update and delete branch and unique date check bug fix.

**To Reproduce**
1. Meeting page
2. Create a date
3. Try to update

**Expected behavior**
Date updates

### PR DESCRIPTION
**Pull request by archlich03**
`app/Http/Controllers/DateController.php`
- Added extra authentification checks

`app/Http/Controllers/MeetingController.php`:
- sorted the meeting dates by `date_and_time` in ascending order.

`app/Http/Requests/StoreDate.php`
- simple bug fix.

`app/Http/Requests/StoreMeeting.php`:
- Changed the value for the max amount of days a meeting can be stored. Storing a meeting for 87+ years is a big no no!

`resources/views/meetings/add.blade.php` and `resources/views/meetings/edit.blade.php`:
- changed text. self explanatory.

`resources/views/meetings/meeting.blade.php`
- added text which would mention when the meeting would end for each date.
- repositioned the deletion button
- added more auth checks

### CODE REVIEW

**Review by Marsietis — APPROVED** [review]
Looks, good. A few notes:
- Very big branch of multiple unrelated things. Could have been better if each smaller issue had their own branch. 
- Meeting page will be almost completly changed in votes branch, so many merge commits might appear, so maybe it is good to remove that commit only.
- Date update fix should have definetly be it's own PR.
- Also unsure about DateController check. I think we already have checks for that.

**Comment by Marsietis** [comment]
Don't merge until votes PR is merged.

**Review by nitrogency — APPROVED** [review]
Looks good, lots of small but needed changes. The authentication checks may add greater security, nice!

**Pull request merged**
